Week5/Blockbuster/app/src/main/java/com/example/blockbuster/MyPrefs.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.blockbuster
+
+import android.content.Context
+import android.content.SharedPreferences
+
+
+class MyPrefs(var context: Context) {
+    //companion object{
+        //var userLogged : Boolean = false
+    //}
+
+    fun savePrefs(userLogged : Boolean){
+        val prefs: SharedPreferences =context.getSharedPreferences("MyPrefs", Context.MODE_PRIVATE)
+        val editor = prefs.edit()
+        editor.putBoolean("isUserLogged",userLogged)
+        editor.apply()
+    }
+
+
+    fun getPrefs() : Boolean{
+        val prefs: SharedPreferences =context.getSharedPreferences("MyPrefs", Context.MODE_PRIVATE)
```

Comment:
You've done a nice job implementing the shared preferences and saving the state.